Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -1638,6 +1638,434 @@ script:
+    outputs:
+    - contextPath: PrismaCloudCompute.DefenderSettings.AdmissionControlEnabled
+      description: The admission control setting.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.DefenderSettings.AdmissionControlWebhookSuffix
+      description: The webhook suffix.
+      type: String
+    - contextPath: PrismaCloudCompute.DefenderSettings.AppEmbeddedFileSystemTracingEnabled
+      description: The file tracing setting.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.DefenderSettings.AutomaticUpgrade
+      description: The automatic upgrade setting.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.DefenderSettings.DisconnectPeriodDays
+      description: The disconnect period in days.
+      type: Number
+    - contextPath: PrismaCloudCompute.DefenderSettings.HostCustomComplianceEnabled
+      description: The custom compliance setting.
+      type: Boolean
+    - contextPath: PrismaCloudCompute.DefenderSettings.ListeningPort
+      description: The defender listening port.
+      type: Number
+  - arguments:
+    - description: The defender hostname.
+      name: hostname
+    - description: The number of log lines to fetch.
+      name: lines
+    description: Download the defender logs.
+    name: prisma-cloud-compute-logs-defender
+    outputs:
+    - contextPath: PrismaCloudCompute.Defenders.Hostname
+      description: The hostname the log was retrieved from.
+      type: String
+    - contextPath: PrismaCloudCompute.Defenders.Logs.Level
+      description: The log level.
+      type: String
+    - contextPath: PrismaCloudCompute.Defenders.Logs.Log
+      description: The log message.
+      type: String
+    - contextPath: PrismaCloudCompute.Defenders.Logs.Time
+      description: The time of the log.
+      type: Date
+  - arguments:
+    - description: The defender hostname.
+      name: hostname
+    - description: The number of log lines to fetch.
+      name: lines
+    description: Download a zip of all defender logs.
+    name: prisma-cloud-compute-logs-defender-download
+    outputs:
+    - contextPath: File.EntryID
+      description: The EntryID of the log bundle.
+      type: String
+    - contextPath: File.Extension
+      description: The extension of the log bundle.
+      type: String
+    - contextPath: File.Info
+      description: The file info.
+      type: String
+    - contextPath: File.MD5
+      description: The MD5 of the log bundle.
+      type: String
+    - contextPath: File.Name
+      description: The name of the log bundle.
+      type: String
+    - contextPath: File.SHA1
+      description: The SHA1 of the log bundle.
+      type: String
+    - contextPath: File.SHA256
+      description: The SHA256 of the log bundle.
+      type: String
+    - contextPath: File.SHA512
+      description: The SHA512 of the log bundle.
+      type: String
+    - contextPath: File.SSDeep
+      description: The SSDeep of the log bundle.
```

Comment:
please change to InfoFile according to context standard https://xsoar.pan.dev/docs/integrations/context-standards-mandatory#infofile

instead of https://xsoar.pan.dev/docs/integrations/context-standards-mandatory#file

Reply:
@nericksen please make sure to handle it please